src/main/geometries/Geometries.java

```diff
@@ -0,0 +1,50 @@
+package geometries;
+
+import java.util.LinkedList;
+import java.util.List;
+import primitives.Point;
+import primitives.Ray;
+
+public class Geometries implements Intersectible {
+
+	private List<Intersectible> intersectibles = new LinkedList<>();
+
+	/**
+	 * Construct a collection of geometries given an array of {@link Intersectible}s or given any number of
+	 * {@link Intersectible}s.
+	 *
+	 * @param intersectibles The {@link Intersectible}s to initialise the collection to.
+	 */
+	public Geometries(Intersectible... intersectibles) {
+		add(intersectibles);
+	}
+
+	/**
+	 * Add a single {@link Intersectible} to the collection.
+	 *
+	 * @param intersectible The {@link Intersectible} to add.
+	 */
+	public void add(Intersectible intersectible) {
```

Comment:
I think this is superfluous. Can't the next function with ellipses accept one argument as well?

Reply:
The one with ... Must construct a new array in the background every time is called. Therefore I separated the most common case (one parameter) into a more efficient function